### ISSUE TITLE
False Positive on Lists in Frontmatter [MD005]

### ISSUE THREAD

**Issue by 0xdf223** [rvben/rumdl#35]
It seems that when there's a list in the front matter _and_ a list later in the MD, it still false raises MD005:

```md
---
layout: post
title: "title"
creator:  
  - 'user1'
  - 'user2'
creator_num:  
  - 1253217
  - 1615089
tags: [tag1, tag2, tag3]
---

# TITLE

## Heading

Whatever

And a list: 

- Item1
- Item2
```

This flags:

```
$ rumdl check -e md005 test9.md 
test9.md:5:1: [MD005] Expected indentation of 0 spaces, found 2 [*]
test9.md:6:1: [MD005] Expected indentation of 0 spaces, found 2 [*]
test9.md:8:1: [MD005] Expected indentation of 0 spaces, found 2 [*]
test9.md:9:1: [MD005] Expected indentation of 0 spaces, found 2 [*]

Issues: Found 4 issues in 1 file (7ms)
Run with `--fix` to automatically fix 4 of the 4 issues
$ rumdl -V
rumdl 0.0.109
```

**Comment by 0xdf223** [comment]
this seems to be resolved as of 0.0.111

**Comment by rvben** [comment]
Thanks for the confirmation!
